network-store-server/src/main/java/com/powsybl/network/store/server/migrations/PermanentLimitsMigration.java

```diff
@@ -0,0 +1,116 @@
+/*
+  Copyright (c) 2024, RTE (http://www.rte-france.com)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.store.server.migrations;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.Lists;
+import com.powsybl.iidm.network.LimitType;
+import com.powsybl.network.store.model.ResourceType;
+import com.powsybl.network.store.server.PermanentLimitSqlData;
+import com.powsybl.network.store.server.dto.OwnerInfo;
+import com.powsybl.network.store.server.dto.PermanentLimitAttributes;
+import liquibase.change.custom.CustomSqlChange;
+import liquibase.database.Database;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.exception.CustomChangeException;
+import liquibase.exception.DatabaseException;
+import liquibase.exception.SetupException;
+import liquibase.exception.ValidationErrors;
+import liquibase.resource.ResourceAccessor;
+import liquibase.statement.SqlStatement;
+import liquibase.statement.core.InsertStatement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.*;
+
+/**
+ * @author Etienne Lesot <etienne.lesot at rte-france.com>
+ */
+public class PermanentLimitsMigration implements CustomSqlChange {
+    private static final Logger LOGGER = LoggerFactory.getLogger(PermanentLimitsMigration.class);
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    @Override
+    public SqlStatement[] generateStatements(Database database) throws CustomChangeException {
```

Comment:
I have an error when trying to migrate a recollement

Reply:
fix